src/core/dfn-panel.js

```diff
@@ -0,0 +1,161 @@
+// Constructs "dfn panels" which show all the local references to a dfn and a
+// self link to the selected dfn. Based on Bikeshed's dfn panels at
+// https://github.com/tabatkins/bikeshed/blob/ef44162c2e/bikeshed/dfnpanels.py
+import { fetchAsset } from "./text-loader.js";
+import { hyperHTML } from "./import-maps.js";
+import { norm } from "./utils.js";
+
+const cssPromise = loadStyle();
+
+export const name = "core/dfn-panel";
+
+export async function run() {
+  const css = await cssPromise;
+  document.head.insertBefore(
+    hyperHTML`<style>${css}</style>`,
+    document.querySelector("link")
+  );
+
+  document.body.addEventListener("click", event => {
+    /** @type {HTMLElement} */
+    const el = event.target;
```

Comment:
```suggestion
    const { target: el } = event;
```

Reply:
Would've to change JSDoc definition this way

Reply:
It actually makes the line longer, what would be the reason to prefer the latter?